packages/core/src/App/Components/Elements/Modals/mul-cfd-upgrade-modal.jsx

```diff
@@ -272,8 +280,10 @@ UpgradeAccountsModal.propTypes = {
     toggleUpgradeAccounts: PropTypes.func,
     openRealAccountSignup: PropTypes.func,
```

Comment:
can we add 'demo_mt5_accounts' and 'virtual_accounts'  to propsTypes here?

Reply:
done